src/vs/base/node/processes.ts

```diff
@@ -50,7 +50,7 @@ function terminateProcess(process: cp.ChildProcess, cwd?: string): Promise<Termi
 			if (cwd) {
 				options.cwd = cwd;
 			}
-			const killProcess = cp.execFile('taskkill', ['/T', '/F', '/PID', process.pid.toString()], options);
+			const killProcess = cp.execFile('taskkill', ['/T', '/F', '/PID', process.pid!.toString()], options);
```

Comment:
There is no behavior change in this version, documentation was updated to reflect the correct behavior and types were updated accordingly https://github.com/nodejs/node/commit/a3c564bead

`pid` will be undefined only when the child process fails to launch and the process is not connected to a shell. Feel free to update this utility for the case when `pid` is undefined and that change can target `main` branch today instead of changing in this PR.